### PR TITLE
Meta: use CSS `:is()`

### PR DESCRIPTION
**Pull request by yakov116**
Cherry-picked out of #4037

I did not see any JS files worth touching now. (maybe after we drop refresh)

### CODE REVIEW

File: source/features/hide-obvious-tooltips.css [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-.notification-indicator::before,
-.notification-indicator::after {
+:is(
+[aria-label^='View the whole file'],
+[aria-label^='Change this file'],
+[aria-label='Delete this file'],
+[aria-label='Edit this file'],
+[aria-label='Edit comment'],
+[aria-label='Edit comment'],
+.notification-indicator
```
Comment (fregante) [inline source/features/hide-obvious-tooltips.css:9]: Indentation?

**Review by fregante — APPROVED** [review]

File: source/features/hide-obvious-tooltips.css (hunk 1 above)
Comment (yakov116) [reply source/features/hide-obvious-tooltips.css:9]: Why I put it on a new line? or why did I not add a tab?

**Comment by yakov116** [comment]
Should we wait until after tomorrow's release?

File: source/features/hide-obvious-tooltips.css (hunk 1 above)
Comment (fregante) [reply source/features/hide-obvious-tooltips.css:9]:
```suggestion
	  [aria-label^='View the whole file'],
	  [aria-label^='Change this file'],
	  [aria-label='Delete this file'],
	  [aria-label='Edit this file'],
	  [aria-label='Edit comment'],
	  [aria-label='Edit comment'],
	  .notification-indicator
```

**Comment by fregante** [comment]
@sindresorhus thoughts on the readability of selectors with `:is()`?

**Comment by sindresorhus** [comment]
I think it's a big improvement.

File: source/features/hide-obvious-tooltips.css (hunk 1 above)
Comment (yakov116) [reply source/features/hide-obvious-tooltips.css:9]: Linter does not let

Comment (yakov116) [reply source/features/hide-obvious-tooltips.css:9]: See https://github.com/sindresorhus/refined-github/runs/1999791363

Comment (fregante) [reply source/features/hide-obvious-tooltips.css:9]: Looks like a stylelint problem. Content inside parens should be indented

Comment (fregante) [reply source/features/hide-obvious-tooltips.css:9]: https://github.com/stylelint/stylelint/issues/5181

**Review by fregante — APPROVED** [review]

**Comment by yakov116** [comment]
@fregante when we do the next release we should add **Requires Chrome 88 and above**

**Pull request merged**